skimage/filters/thresholding.py

```diff
@@ -335,15 +337,20 @@ def threshold_li(image):
     >>> thresh = threshold_li(image)
     >>> binary = image > thresh
     """
+    assert_nD(image, 2)
     # Requires positive image (because of log(mean))
-    offset = image.min()
-    # Can not use fixed tolerance for float image
-    imrange = image.max() - offset
-    image -= offset
+    offset = img.min()
```

Comment:
Don't call this offset but `immin`, since you will need it later when computing the range.